openapi/components/schemas/Quote.yaml

```
@@ -70,6 +70,20 @@ properties:
           oneOf:
             - $ref: ./Plans/OriginalPlan.yaml
             - $ref: ./Plans/FlexiblePlan.yaml
+        description:
+          type: string
+          description: Description of the quote item.
+          example: Charge per approved transaction with DCC
+          default: ''
+          maxLength: 255
+        priceDescription:
+          type: string
+          description: |-
+            Price description of the quote item.
+            This value is used only with metered billing items.
```

suggestion:: for readability
```suggestion
            This value is only used for metered billing items.
```